app/assets/javascripts/utils/course_date_utils.js

```diff
@@ -50,7 +50,7 @@ const CourseDateUtils = {
       },
       timeline_start: {
         minDate: startDate,
-        maxDate: parseISO(course.timeline_end)
+        maxDate: parseISO(course.end)
```

Comment:
I don't think this is a change we want to make. The timeline start should always be before the timeline end, and the timeline end should always be before the course end.

Reply:
I reverted the changes so that i could debug why the CI workflow is failing. But i'm planning on putting them back.

Reply:
> I don't think this is a change we want to make. The timeline start should always be before the timeline end, and the timeline end should always be before the course end.

Concerning this, a user will start by selecting the timeline start and at that point we are not sure of what date the user will select for timeline end. This means we may not be able to validate the max date for timeline start. Is this what is expected?